<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProfile</class>
 <widget class="QDialog" name="CreateProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:century gothic;
	font-size: 16px;
}


QFrame{
	background:rgba(0, 0, 0, 200);
	border-radius: 15px;
}

#frame_2{
	background:transparent;
}
QPushButton{
	font-size:14px;
	background:green;
	border-radius: 15px;
}

QToolButton{
	background:green;
	border-style: solid;
	color:darkgray;
	font-size:13px;
	border-radius: 60px;
}

QToolButton:hover{
	background:#16a016;
	border-style: solid;
	color:darkgray;
	font-size:13px;
	border-radius: 60px;
}

QLabel{
	color: white;
	background:transparent;
}

QPushButton{
	color:white;
	/* border-radius: 15px; */
}

QPushButton:hover{
	/*color:darkgray;*/
	background-color:#16a016;
	border: black;
	/* border-radius: 15px; */
}

QLineEdit{
	background: transparent;
	border: none;
/*	color: #717072;*/
color:white;
	border-bottom: 1px solid #717072;
	font-size: 18px;

}

QCheckBox{
	font-size:13px;
	background: none;
	color:darkgray;
}

#formLayout{
color:white;
}

#CreateProfile{
	background: url(/home/groumeliotis/project/Sarah/images/background.jpg);
}

#pushButton_forgot_pwd{
	background: transparent;
	font-size: 12px;

}
#pushButton_forgot_pwd:hover{
	background: transparent;
	font-size: 13px;
}

#label_usr{
	font-size:15px;
	color:red;
}
#label_pwd{
	font-size:15px;
	color:red;
}
#label_logo_S{
	font-size:40px;
    font-family: 'Times New Roman', Times, serif;
}

#label_logo_arah{
	font-size:14px;
    font-family: 'Times New Roman', Times, serif;
}

QRadioButton{
	font-size: 14px;
	color: darkgray;
}

QDateEdit{
	font-size: 14px;
	color: darkgray;
}

QSpinBox{
	font-size: 14px;
	color: darkgray;
}

QDoubleSpinBox{
	font-size: 14px;
	color: darkgray;
}
#label_userExist{
	font-size:15px;
	color:red;
}
#label_pnm{
	font-size:15px;
	color:red;
}
#label_gender{
	font-size:15px;
	color:red;
}

#label_weight{
	font-size:15px;
	color:red;
}

#label_height{
	font-size:15px;
	color:red;
}

#label_lname{
	font-size:15px;
	color:red;
}

#label_fname{
	font-size:15px;
	color:red;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="2" alignment="Qt::AlignRight|Qt::AlignBottom">
       <widget class="QPushButton" name="pushButton_submit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_logo_arah">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Meals Per Day</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignRight">
           <widget class="QSpinBox" name="spinBox_meals">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="gluten">
            <property name="text">
             <string> I can consume gluten</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="diabetic">
            <property name="text">
             <string>I can't consume sugar</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="vegan">
            <property name="text">
             <string>I am vegan</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="lact">
            <property name="text">
             <string>I can consume lactose</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="2">
           <widget class="QRadioButton" name="male">
            <property name="text">
             <string>Male</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="female">
            <property name="text">
             <string>Female</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" alignment="Qt::AlignRight">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_height">
            <property name="maximum">
             <double>2.500000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Gender</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Workout Per Week</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3" alignment="Qt::AlignRight">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_weight">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>500.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Weight</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Birthday</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3" alignment="Qt::AlignRight">
           <widget class="QSpinBox" name="spinBox_workout">
            <property name="maximum">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QDateEdit" name="dateEdit_bday">
            <property name="currentSection">
             <enum>QDateTimeEdit::YearSection</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_gender">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_height">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_weight">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_fname">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="placeholderText">
             <string>Firstname</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_lname">
            <property name="placeholderText">
             <string>Lastname</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_userExist">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_username">
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_confirm_password">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Confirm Password</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_pnm">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_fname">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_lname">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="3">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="styleSheet">
             <string notr="true">*{
	background-color:transparent;
}</string>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>323</width>
               <height>68</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QPushButton" name="pushButton_fexce">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Food Exceptions</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QToolButton" name="toolButton_avatar_pic">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../images/avatar.png</normaloff>../../images/avatar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolButton_avatar_pic</tabstop>
  <tabstop>lineEdit_fname</tabstop>
  <tabstop>lineEdit_lname</tabstop>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>lineEdit_confirm_password</tabstop>
  <tabstop>spinBox_meals</tabstop>
  <tabstop>diabetic</tabstop>
  <tabstop>gluten</tabstop>
  <tabstop>vegan</tabstop>
  <tabstop>lact</tabstop>
  <tabstop>dateEdit_bday</tabstop>
  <tabstop>doubleSpinBox_height</tabstop>
  <tabstop>doubleSpinBox_weight</tabstop>
  <tabstop>spinBox_workout</tabstop>
  <tabstop>pushButton_submit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
